boundlessgeo/exchange
# Suppress Documents and Documents Upload if DOCUMENTS_ENABLED env var …

…is False

## JIRA Ticket
GVSD-9129

## Description
Allows Documents to be disabled by setting a custom env var, `DOCUMENTS_ENABLED = False`.

## TODO

## Steps to Test or Reproduce
Set `DOCUMENTS_ENABLED` to `False` and the following should happen:

- Menu options for Data > Documents and Data > Upload Document should disappear
- All /documents urls should give an error message that documents have been disabled
- Documents subsection under profile details page should no longer exist

## Setup Environment with PR

1. Cleanup previous state

```bash
make purge
```

2. Checkout PR

__Using git command (command line interface)__
```bash
pr_id=ADD_PR_NUMBER_HERE
git checkout master
git branch -D $pr_id
git fetch
git fetch origin pull/$pr_id/head:$pr_id
git checkout $pr_id
git submodule init
git submodule update --remote --recursive
```

__Using [Gitkraken](https://www.gitkraken.com/)__

+ In GitKraken, right click on the pull request you want to review. Select Add Remote and Checkout (the PR).

3. Start exchange

```bash
make start
```

4. Exchange Healthcheck

```bash
docker inspect --format '{{ .State.Health.Status }}' exchange
```

__NOTE:__ Only continue the following steps if the output from the above command is `healthy`. You may have to wait 
a few minutes.

---
@boundlessgeo/bex-qa
